### DISCUSSION TITLE
"Test" and spinning icon

### DISCUSSION THREAD

**Discussion by ryfindling** [davetapley/helium-tax#12] [Q&A] [answered]
When I input my hotspot address it spins saying test and never resolves. Any suggestions? Thank you :)
<img width="97" alt="Screen Shot 2021-02-14 at 12 57 51 PM" src="https://user-images.githubusercontent.com/78979324/107884679-598d2600-6ec4-11eb-96f8-f67e3681fe24.png">

**Comment by jondoeh** [comment]
i see the same :(

**Reply by davetapley** [reply]
I've had people on Twitter say they're getting this too 😞 

@ryfindling and @jondoeh could you let me know what browser / OS you are using? 🙏🏻

**Comment by ryfindling** [comment]
I tried in Chrome and in Safari

**Reply by davetapley** [reply]
Okay, I've added an extra step to check for errors, can you see if it shows up with a ⚠️ message?

**Comment by jondoeh** [comment]
I was using Brave (which is based on the Chrome kernel i believe)
Then realized that might be the issue and switched to Safari with the same
result.

**Reply by davetapley** [reply]
Okay, I've added an extra step to check for errors, can you see if it shows up with a ⚠️ message?

**Reply by davetapley** [reply]
@jondoeh I believe this is now fixed, let me know! 🤞🏻

**Comment by ryfindling** [comment]
When I enter my hotspot address, click do some taxes, it shows the spinning icon for a second, then stops and does nothing. Tried in Safari and Chrome

**Reply by davetapley** [reply]
Mad! 🤪 
Could you send me your hotspot address?
This is public so if you want to email me you can find my email on my profile: https://github.com/davetapley

Thanks for your patience 🙏🏻

**Comment by ryfindling** [comment]
No worries - will shoot you an email. Thank you so so much for all of your help

**Comment by davetapley** [comment, marked as answer]
Presuming this was fixed on #13
